commons/util-el/src/main/java/io/github/microcks/util/el/ExpressionParser.java

```diff
@@ -88,7 +88,12 @@ public static Expression[] parseExpressions(String template, EvaluationContext c
                throw new ParseException(template, prefixIndex, "No expression defined within delimiter '"
                      + expressionPrefix + expressionSuffix + "' at character " + prefixIndex);
             }
-            expressions.add(doParseExpression(expr, context));
+            if (expr.charAt(0) == '{' || expr.charAt(0) == '(' || expr.charAt(0) == '[' || expr.charAt(0) == '#') {
```

Comment:
Looks good but I think we don't have to manage the `(`, `[` or `#` cases. These are already possible using standard notation. See my above comment.

Reply:
Oh! Yeah, valid point. Actually, I was thinking about everything inside `{{}}`. I removed explicit check now.